### PR TITLE
Missing translations smit

### PR DESCRIPTION
**Pull request by smitsanghvi21**

### CODE REVIEW

File: site/nl_BE.all.json [hunk 1]
```diff
@@ -556,43 +553,44 @@
   "donate_button_text": {
     "other": "Doneren"
   },
-  "wcag_skip_links_header": { "other": "Accessibility Links" },
-  "wcag_skip_links_content": { "other": "Skip to Content" },
+  "wcag_skip_links_header": { "other": "Toegankelijkheidslinks" },
+  "wcag_skip_links_content": { "other": "Skip naar Inhoudd" },
+  "wcag_skip_links_footer": { "other": "Ga naar voettekst" },
+  "wcag_skip_links_content": { "other": "Doorgaan naar artike" },
```
Comment (stajs) [inline site/nl_BE.all.json:559]: `wcag_skip_links_content` key appears twice.

**Review by brynyShift72 — CHANGES_REQUESTED** [review]
You've missed some, some translations here may be fine
nl_BE:
`validation_pincode_required`, `shopping_card_existing_expired`,`shopping_card_existing_expiry`, `shopping_card_update_reason_none`, `availability_expired`,  `availability_expires`
no_NO:
`signin_form_error_ip_throttled`,`signin_form_error_account_suspended`, `shopping_discount_applied`, `shopping_error_discount_worn_out`, `shopping_card_existing_expiry`, `shopping_card_existing_expired`, 
`pl_PL`:
`shopping_card_existing_expiry`, `shopping_card_existing_expired` For these 3, you can find the English version in the newest en_AU: `shopping_card_update_reason_none`, `shopping_info_ownership_plan`, `shopping_action_credit`
pt_BR:
`account_form_pin_code_not_set`, `account_form_pin_changed`,`account_form_change_pin`,`user_set_pin_header`,`user_set_pin_button`, `user_submit_pin_heading`, `user_error_wrong_pin_retry`, `user_submit_pin_button`, `user_reset_pin_button`, `validation_pincode2_match`, `validation_pincode2_pattern`, `validation_pincode1_pattern`, `validation_pincode2_required`, `validation_pincode1_required`, `shopping_payment_method_header`, `shopping_payment_method_credit_card`, `shopping_error_stripe_unknown_error`, `shopping_error_payment_complete_failed`, `availability_expires`, `availability_expired`
`pt_PT `Looks good to go

**Review by curtisadams28 — CHANGES_REQUESTED** [review]
`wcag_skip_links_footer` Is no longer being used so should be removed.

File: site/pt_BR.all.json [hunk 2]
```diff
@@ -1237,12 +1150,7 @@
...
 
-  "shopping_error_plan_already_owned": { "other": "You already have this plan." },
-  "shopping_error_plan_expired": { "other": "This plan is no longer available." },
-
-  "plan_label_owned": { "other": "" },
-
-  "wcag_aria_label_letterboxd": { "other": "View on Letterboxd" }
-
+  "shopping_error_plan_already_owned": { "other": "Você já tem este plano." },
+  "shopping_error_plan_expired": { "other": "Este plano não está mais disponível." },
```
Comment (curtisadams28) [inline site/pt_BR.all.json:1154]: Trailing comma

**Review by curtisadams28 — CHANGES_REQUESTED** [review]

**Review by curtisadams28 — APPROVED** [review]
LGTM

**Review by brynyShift72 — APPROVED** [review]
LGTM too

**Pull request merged**